.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release - Publish to PyPI
+
+on:
+  release:
+    types: [published]
```

Comment:
The action will the triggered only once the release is published (so, for example, the draft state will not trigger it).